uint16 id
uint8 hamming_error
geometry_msgs/PoseWithCovarianceStamped pose
float32[9] orientation
float32[3] translation
float32[8] corners